app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  attr_accessor :remember_token
```

Comment:
không dùng attr_accessor
```suggestion
  attr_reader :remember_token
```